code/src/components/TheLabyrinth.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { fetchLabyrinthPath } from 'reducers/labyrinth'
+import { PositionIcon } from './PositionIcon'
+import { Footer} from './Footer'
+import { RestartBtn } from './RestartBtn'
+import { 
+    H3,
+    H2,
+    ContentContainer,
+    DirectionBtn,
+    BackgroundImg,
+    P,
+    Coordinates,
+    PositionContainer,
+    TheCardContainer,
+    CardItemContainer,
+    EndingText
+    } from '../lib/styling' 
+
+
+
+export const TheLabyrinth = () => {
+    const dispatch = useDispatch()   
+    const { description, coordinates, actions} = useSelector(store=>  store.labyrinth.currentPosition)
+    
+    const handleButtonOnClick = (type, direction) => {
+            dispatch(fetchLabyrinthPath(type, direction))
+        }
+
+        const TheCard = ({ description, type, direction }) => (
+                <CardItemContainer>
+                    <P>{description}</P>
+                    <DirectionBtn onClick={() => handleButtonOnClick(type, direction)}>
+                    {type.toUpperCase()} {direction.toUpperCase()} <span> ?</span>
+                    </DirectionBtn>
+                </CardItemContainer>
+        )
+
+        const imgLibrary = {
```

Comment:
Nice way to organize the pictures in a library, Changing background picture make the player feel like they are moving forward.